docs/source/contributing/testing.rst

```diff
@@ -26,30 +29,28 @@ At the moment there are three type of tests:
 
 #. Unit Tests:
 
-   Basically test for pretty much everything. For example, there a test for
+   Tests for most of the basic functionality of manim. For example, there a test for
    ``Mobject``, that checks if it can be added to a Scene, etc ..
 
 #. Graphical unit tests:
 
-   Because ``manim`` is a video library, we tests frames. To do so, we take a
-   frame of control data for each feature and compare the last frame of the
-   feature rendered (in the form of a numpy array). If it matches, the tests
-   are successful. If one wants to visually see the what has changed, you can
-   use ``--show_diff`` flag along with ``pytest`` to be able to visualize
-   what is different.
+   Because ``manim`` is a graphics library, we test frames. To do so, we create test scenes that render a specific feature.
+   When pytest runs, it compares the last frame of every render to the control data; If it matches, the tests
+   pass. If you want visually see the what has changed, you can
+   use ``--show_diff`` flag with ``pytest``.
```

Comment:
It's not accurate, --show_diff is triggered _only_ when a the control data and the generated frame differ.